packages/updating-element/src/decorators/queryAssignedNodes.ts

```diff
@@ -80,6 +80,7 @@ export function queryAssignedNodes(
           nodes = nodes.filter(
             (node) =>
               node.nodeType === Node.ELEMENT_NODE &&
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Free bytes: remove the whole check!

https://developer.mozilla.org/en-US/docs/Web/API/Element/matches#Browser_compatibility

Reply:
IE11 uses the prefix.

Reply:
argh... can we push this to the polyfills?

Reply:
Definitely: https://github.com/webcomponents/polyfills/issues/397